lib/orderbook/OrderBook.ts

```diff
@@ -298,6 +298,17 @@ class OrderBook extends EventEmitter {
       };
     }
 
+    // check if sufficient outbound channel capacity exists
+    const { makerAmount } = Swaps.calculateSwapAmounts(order.quantity, order.price, order.isBuy);
+    const { makerCurrency } = Swaps.deriveCurrencies(order.pairId, order.isBuy);
+    const swapClient = this.swaps && this.swaps.swapClients[makerCurrency];
```

Comment:
For getting the tests to pass, how about wrapping this block of code in an `if (this.swaps)` check and dropping it from this line?

Reply:
This won't work because the service spec is firing up a real instance of xud (with config defined in the test setup). It will have `this.swaps`.

Reply:
I've updated this PR/commit to make sanity checks configurable from xud.conf (`sanitychecks = true`). This allows us to disable those checks in the service spec.

Alternative option is to remove the offending tests from service spec and add them to xud-simulation... 🤔